AppCenterAuth/AppCenterAuth/Internals/Models/MSAADAuthority.m

```diff
@@ -0,0 +1,56 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#import "MSAADAuthority.h"
+#import "../Util/MSAuthConstants.h"
+
+@implementation MSAADAuthority
+
+static NSString *const kMSAudienceType = @"type";
+
+static NSString *const kMSAuthorityUrl = @"authority_url";
```

Comment:
Lots of these constants are keys so they should indicate that:

```suggestion
static NSString *const kMSAuthorityUrlKey = @"authority_url";
```

Reply:
changed constants to keys